dd-sdk-android/src/main/kotlin/com/datadog/android/instrumentation/gesture/DatadogGesturesTracker.kt

```diff
@@ -0,0 +1,100 @@
+package com.datadog.android.instrumentation.gesture
+
+import android.app.Activity
+import android.view.GestureDetector
+import android.view.MotionEvent
+import androidx.core.view.GestureDetectorCompat
+import com.datadog.android.log.Logger
+
+internal class DatadogGesturesTracker(val uiInteractionLogger: Logger) : GesturesTracker,
```

Comment:
`uiInteractionLogger` - will this be used by `Window.Callback` later? Or why is it here?

Reply:
yup ....this is in progress....I will let you know once this is finished to have another review.